includes/Integrations/Jetpack.php

```diff
@@ -376,7 +377,7 @@ public function add_term( $mid, $object_id, $meta_key ): void {
 			return;
 		}
 
-		wp_set_object_terms( (int) $object_id, 'poster-generation', $this->media_source_taxonomy->get_taxonomy_slug() );
+		wp_set_object_terms( (int) $object_id, Media_Source::POSTER_GENERATION, $this->media_source_taxonomy->get_taxonomy_slug() );
```

Comment:
Again here, if `Media_Source` and `Media_Source_Taxonomy` were merged, we this could be 
```
wp_set_object_terms( (int) $object_id, $this->media_source_taxonomy::POSTER_GENERATION, $this->media_source_taxonomy->get_taxonomy_slug() );
```